app/controllers/experiments/list.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Controller.extend({
     isShowingModal: false,
     breadCrumb: 'Experiments',
     queryParams: ['sort', 'match', 'state', 'q'],
-    state: null,
+    state: 'All',
```

Comment:
I know this is only one usage of existing code, but...

In general, one common failsafe to deal with (possibly case-sensitive) fields is to see if `value.upper() == UPPERCASE_INTERNAL_VALUE`. But in order for that to work, the allowed values of `state` probably shouldn't be mixed-case.

(eg prefer `ACTIVE` to `Active`)